ps-modules/MacUtil/MacUtil.psm1

```diff
@@ -0,0 +1,202 @@
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
+
+function ConvertTo-UniversalBinaries {
+    param (
+        [string]$arm64Dir,
+        [string]$x64Dir,
+        [string]$universalDir
+    )
+
+    Write-Host "Creating universal dir and copying headers..."
+    if (Test-Path -Path $universalDir -PathType Container) {
+       Remove-Item -Path "$universalDir" -Recurse -Force | Out-Null
+    }
+    New-Item -Path "$universalDir" -ItemType Directory -Force | Out-Null
+    $arm64LibDir = Join-Path $arm64Dir "lib"
+    $x64LibDir = Join-Path $x64Dir "lib"
+    Copy-Item -Path "$x64Dir\include" -Destination "$universalDir\include" -Recurse | Out-Null # Assume arm64 and x86_64 are identical
```

Comment:
the backslashes here look wrong, especially since this is inside of MacUtil?

Reply:
Fixed: 1e88828f510b8c2053c76536ad6d7396eb50af01